src/util/validate.ts

```diff
@@ -0,0 +1,10 @@
+import { Identity } from '../identity';
+
+export function validateIdentity(identity: Identity): string {
+    let errorMsg;
+
+    if (typeof identity !== 'object' || typeof identity.uuid !== 'string') {
```

Comment:
can we check that if a name is present that it is a string, this eliminates the following class inconsistency: 
```js
fin.Application.wrapSync({name: 123, uuid:'123'}) // totally fine
fin.Application.start({name: 123, uuid:'123', url: location.href}) // this tosses an error
```

Reply:
'start' function uses app options object, not identity.